events/events-team/content/new-contributor-workshops/build-and-test.md

```diff
@@ -0,0 +1,27 @@
+## Build and Test
+
+### Duration
+
+40-60 minutes, depending on beginner vs intermediate participants
+
+### Task Overview
+
+Build Kuberentes binaries locally, and do some fun things with them.
+
+**Important** Have facilitators to help with technical issues (2-3 is enough for Intermediate, you want 5 for Beginner)
+
+### Slides
+
+Clear command line instructions.
+
+### Details To Cover 
+
+- This is the hands-on part. Details are up to you; look to previous workshops for inspiration.
+- Adjust difficulty based on whether this is the Beginner or Intermediate workshop.
+- It is _extremely important_ to run this part of the workshop several times previously, and to make sure that versions match those in the [workspace setup](./workspace-setup.md).
+
```

Comment:
Add small breaks between sets of instructions for proctors to go and help people with issues, This ensures no one gets left behind.